preview/Win7Msix/Win7MSIXInstaller/MsixRequest.cpp

```diff
@@ -46,15 +47,16 @@ struct HandlerInfo
 std::map<PCWSTR, HandlerInfo> AddHandlers =
 {
     //HandlerName                         Function to create                      NextHandler                          ErrorHandlerInfo
-    {PopulatePackageInfo::HandlerName,    {PopulatePackageInfo::CreateHandler,    CreateAndShowUI::HandlerName,        ErrorHandler::HandlerName}},
-    {CreateAndShowUI::HandlerName,        {CreateAndShowUI::CreateHandler,        ProcessPotentialUpdate::HandlerName, ErrorHandler::HandlerName}},
+    {PopulatePackageInfo::HandlerName,    {PopulatePackageInfo::CreateHandler,    CreateAndShowUI::HandlerName,        nullptr}},

```

Comment:
why is this now nullptr? It might be good to put inline comments to clarify.

Reply:
i'll need to merge this in with Rudy's split change, but the reasoning here is that ErrorHandler (which does rollback) expects packageInfo to be filled in. early failures to open the package should not go to ErrorHandler.


---
In reply to: [277068659](https://github.com/Microsoft/msix-packaging/pull/87#discussion_r277068659) [](ancestors = 277068659)